strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/repositories/RepositoriesTestConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.carlspring.strongbox.repositories;
+
+import org.carlspring.strongbox.StorageApiTestConfig;
+import org.carlspring.strongbox.config.DataServiceConfig;
+import org.carlspring.strongbox.config.hazelcast.HazelcastInstanceId;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Primary;
+
+@Import({ DataServiceConfig.class, StorageApiTestConfig.class })
+public class RepositoriesTestConfig
+{
+
+    @Primary
+    @Bean
+    public HazelcastInstanceId hazelcastInstanceIdAcctit()
+    {
+        return new HazelcastInstanceId("RepositoriesTestConfig-hazelcast-instance");
+    }
```

Comment:
I thought we weren't currently using Hazelcast for anything in the code on your branch. Is this, or is this not the case?

Reply:
we using hazelcast, you could check `org.carlspring.strongbox.data.CacheName` to see how exactly it used